config_defaults.yml

```diff
@@ -533,6 +533,8 @@ Gemfile:
         require: false
         platforms:
           - ruby
+      - gem: 'parallel_tests'
```

Comment:
This is already being pulled in by the default gemset. Did you have a situation where this didn't happen?

Reply:
I'm pretty sure I tried to run parallel_spec and it gave me an error without it added but I can test again when I rebase this.

Reply:
Yea, looks like parallel_tests is there by default so I removed that addition to the Gemfile.